WordPress/Classes/Services/BlogService.m

```diff
@@ -744,6 +734,27 @@ - (NSTimeZone *)timeZoneForBlog:(Blog *)blog
     return timeZone;
 }
 
+- (void)updateBlog:(Blog *)blog settingsWithRemoteSettings:(RemoteBlogSettings *)remoteSettings
+{
+    blog.blogName = remoteSettings.name;
+    blog.blogTagline = remoteSettings.desc;
+    if (remoteSettings.defaultCategory) {
```

Comment:
@jleandroperez  Only siteVisibility will not hurt, for the other two I'm allowing the use to change them locally even if they cannot being synced with the server. This allows them to define locally what are the default for the local posts, and not have them to set for every post they write.